### ISSUE TITLE
setting crashes while setting the any kind of lockscreen

### ISSUE THREAD

**Issue by mohammad556655** [TrebleDroid/treble_experimentations#47]
Hi 
after installing with twrp whenever i want to set lock screen, suddenly it goes back previous step and the setting crash will prompt. then if i restart the device, i will stuck in a somethin like a bootloop. device will boot but the it loops between splash and lockscreen. and i can't get into my phone. i don't have any problem with android 11 gsi, i get this bug when i use android 12 or 13 gsi. 
i hope this fixes. thank you

**Comment by phhusson** [comment]
Does it mean that this issue doesn't happen when flashing it with stock
recovery?

Le mar. 11 juil. 2023 à 05:56, mohammad556655 ***@***.***> a
écrit :

**Comment by mohammad556655** [comment]
No, does not matter which recovery i use, stock or twrp, this issue will remain. just for android 12 and android 13 . android 11 and android10 are okay.  i installed several gsis all have the same issue with android 12 and 13.
